blocks/bricks/conv.py

```diff
@@ -177,6 +180,144 @@ def num_output_channels(self):
         return self.num_filters
 
 
+class ConvolutionalTranspose(Initializable):
+    """Performs the transpose of a 2D convolution.
+
+    Parameters
+    ----------
+    filter_size : tuple
+        The height and width of the filter (also called *kernels*).
+    num_filters : int
+        Number of filters per channel.
+    num_channels : int
+        Number of input channels in the image. For the first layer this is
+        normally 1 for grayscale images and 3 for color (RGB) images. For
+        subsequent layers this is equal to the number of filters output by
+        the previous convolutional layer. The filters are pooled over the
+        channels.
+    original_image_size : tuple
+        The height and width of the output (image or feature map).
+    batch_size : int, optional
+        Number of examples per batch. If given, this will be passed to
+        Theano convolution operator, possibly resulting in faster
+        execution.
+    image_size : tuple, optional
+        The height and width of the input (image or feature map). If given,
+        this will be passed to the Theano convolution operator, resulting
+        in possibly faster execution times.
+    step : tuple, optional
+        The step (or stride) with which to slide the filters over the
+        image. Defaults to (1, 1).
```

Comment:
This has a particular strange meaning in the case of this brick. Again, it's the stride of the original un-transposed convolution.